sorc/ncep_post.fd/MDL2P.f

```diff
@@ -1816,6 +1819,101 @@ SUBROUTINE MDL2P(iostatusD3D)
           ENDIF
         ENDIF
 !     
+!***  CHIPSI
+!
+        IF (IGET(1021) > 0 .or. IGET(1022) > 0) THEN
```

Comment:
@JesseMeng-NOAA @KarinaAsmar-NOAA This new scheme will only applied in global applications. Please change as:
```
IF ( (IGET(1021) > 0 .or. IGET(1022) > 0). and. MODELNAME == 'GFS' ) THEN
```